streams/ir.m3u

```diff
@@ -79,3 +79,5 @@ https://livefa.marjaeyattv.com/mtv_fa/playlist.m3u8
 http://cdnlive.irib.ir/live-channels/smil:sahar3/playlist.m3u8
 #EXTINF:-1 tvg-id="",Toonix (720p) [Not 24/7]
 http://51.210.227.134/hls/stream.m3u8
+#EXTINF:-1 tvg-id="",4U TV (720p)
```

Comment:
According to the database, this is the Turkish channel: https://iptv-org.github.io/channel?id=4UTV.tr

```suggestion
#EXTINF:-1 tvg-id="4UTV.tr",4U TV (720p)
```

Should be placed in `streams/tr.m3u`